specification/iothub/resource-manager/Microsoft.Devices/preview/2019-03-22-preview/iothub.json

```diff
@@ -2534,6 +2541,33 @@
         "isEnabled"
       ]
     },
+    "EnrichmentProperties": {
+      "description": "The properties of an enrichment that your IoT hub applies to messages to endpoints.",
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "The key for the enrichment property.",
+          "type": "string"
+        },
+        "value": {
+          "description": "The value for the enrichment property.",
+          "type": "string"
+        },
+        "endpointNames": {
+          "description": "The list of endpoints for which the enrichment property is applied to the message if the message satisfies the routing rules for those endpoints.",
```

Comment:
consider deleting "if the message satisfies routing rules...….." , because sometimes routing may not be used, when sending messages to built-in endpoint.

Reply:
yeah I agree, when sending messages sounds better